sycl/plugins/unified_runtime/pi2ur.hpp

```diff
@@ -2697,6 +2693,43 @@ inline pi_result piMemBufferCreate(pi_context Context, pi_mem_flags Flags,
   ur_buffer_properties_t UrProps{};
   UrProps.stype = UR_STRUCTURE_TYPE_BUFFER_PROPERTIES;
   UrProps.pHost = HostPtr;
+
+  ur_buffer_channel_properties_t bufferChannelProperties{};
+  bufferChannelProperties.stype = UR_STRUCTURE_TYPE_BUFFER_CHANNEL_PROPERTIES;
+  ur_buffer_alloc_location_properties_t bufferLocationProperties{};
+  bufferLocationProperties.stype =
+      UR_STRUCTURE_TYPE_BUFFER_ALLOC_LOCATION_PROPERTIES;
+  if (properties != nullptr) {
+    constexpr uint64_t defaultBufferLocation = 7777;
+    constexpr uint32_t defaultBufferMemChannel = 7777;

```

Comment:
I understand that this is astronomically unlikely to cause a problem by being a legit value for these properties one day, but I still think it's asking for trouble. I'd rather these two were boolean flags that get set if we find the corresponding property

Reply:
Thanks @aarongreig ,

I have updated accordingly.